### DISCUSSION TITLE
"Configuring Neofetch to Run Only Once on Terminal Launch in i3wm"

### DISCUSSION THREAD

**Discussion by KIRAN-KUMAR-K3** [theonlyanson/Custom-setup-config-for-i3wm-users#7] [Ideas]
I added the `neofetch` executable command in `nano ~/.config/i3/config`.

```bash
set $ran_neofetch 0

exec_always --no-startup-id [ $ran_neofetch -eq 0 ] && neofetch && export ran_neofetch=1
```

After saving it and pressing Win+Shift+R to restart i3wm settings, I restarted the system. However, it still did not execute when I opened the terminal after the restart.

**Comment by theonlyanson** [comment]
you should update your .bashrc (if u are using bash terminal) or .zshrc (if you are using zsh terminal). Just type 
```bash
nano .bashrc
```
then type
```bash
neofetch
```
and save it...

You cannot access terminal settings using i3wm config

**Reply by KIRAN-KUMAR-K3** [reply]
i updated it, still not executing on startup ,only once when the system is restarted and opens terminal for first time

**Reply by theonlyanson** [reply]
have you installed neofetch ??

**Reply by KIRAN-KUMAR-K3** [reply]
"Yes, I have installed it, but unfortunately, my concept doesn't seem to work in that terminal. Thank you for your cooperation."
